### PR TITLE
Feat:Added Home Page

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat:Added Home Page

landing page with a list of featured categories.
If clicked on any one of the categories it will be redirected to the product list page with the selected categories.

### CODE REVIEW

File: Home.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+}
+h4{
+    margin-top: -0.7rem;
+    text-align: center;
+    color: var(--black-color);
+}
+.anchor-link{
+    text-decoration: none;
+}
+h2{
```
Comment (tusharkandpal) [inline Home.css:58]: avoid using styles on elements directly!

File: Home.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    margin-top: -1.5rem;
+}
+.red-shoe-heading{
+    margin-top: -1.3rem;
+}
+.black-shoe-heading{
+    margin-top: 1rem;
+}
+/* css for footer */
+footer{
```
Comment (tusharkandpal) [inline Home.css:71]: here too

File: index.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                    <div class="outer-img-div"><img class="inner-img-div" src="Image/shoe10.webp" alt="Italian Trulli">
+                        <a class="anchor-link" href="/E-commerce-items/product-list-page.html"><h4 >Upto 25% off</h3></a>
+                    </div> 
+                </div><br><br><br>
+                <!-- footer -->
+                <footer>
+                    <div class="footer-content">
+                        <h2>Made by Neetu kumari</h2>
+                        <ul class="social-media">
+                            <a href="https://github.com/neetukumari4858/E-commerce"><i class="fa fa-github fa_custom" style='font-size:24px'></i></a>
```
Comment (tusharkandpal) [inline index.html:77]: instead of inline style, you can create a class for it

**Review by tusharkandpal — APPROVED** [review]
suggested few changes, rest is ok! good work 💯

**Review by Manju-sree — CHANGES_REQUESTED** [review]
other than request changes from above reviews.. please add gif or previews links before raising PR for review.

**Review by shaksham08 — APPROVED** [review]
Looks good just follow the comment which tushar has given

**Pull request merged**